.github/scripts/generate-packages-linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
```

Comment:
[It'd be safer](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/) to add at least `set -e`, so that the script will exit if any command fails. With the suggestion below, `set -o pipefail` is necessary too. And we can add `set -u` just in case.
```suggestion
#!/usr/bin/env bash
set -euo pipefail
```